java/configuration/src/main/java/sleeper/configuration/properties/instance/EMRServerlessProperty.java

```diff
@@ -0,0 +1,117 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sleeper.configuration.properties.instance;
+
+import sleeper.configuration.properties.SleeperPropertyIndex;
+
+import java.util.List;
+
+public interface EMRServerlessProperty {
+
+    UserDefinedInstanceProperty BULK_IMPORT_EMR_SERVERLESS_CLASS_NAME = Index
+            .propertyBuilder("sleeper.bulk.import.emr.serverless.class.name")
+            .description("The class to use to perform the bulk import.")
+            .propertyGroup(InstancePropertyGroup.BULK_IMPORT)
+            .defaultValue(
+                    "sleeper.bulkimport.job.runner.dataframelocalsort.BulkImportDataframeLocalSortDriver")
+            .runCDKDeployWhenChanged(true).build();
+    UserDefinedInstanceProperty BULK_IMPORT_EMR_SERVERLESS_ARCHITECTURE = Index
+            .propertyBuilder("sleeper.bulk.import.emr.serverless.architecture")
+            .description("The architecture for EMR Serverless to use. X86_64 or ARM (Coming soon)")
+            .propertyGroup(InstancePropertyGroup.BULK_IMPORT).defaultValue("X86_64")
+            .runCDKDeployWhenChanged(true).build();
+    UserDefinedInstanceProperty BULK_IMPORT_EMR_SERVERLESS_ENABLED = Index
+            .propertyBuilder("sleeper.bulk.import.emr.serverless.enabled")
+            .description("The switch to enable EMR Serverless over persistent EMR.")
+            .propertyGroup(InstancePropertyGroup.BULK_IMPORT).defaultValue("true")
+            .runCDKDeployWhenChanged(true).build();
+    UserDefinedInstanceProperty BULK_IMPORT_EMR_SERVERLESS_RELEASE = Index
+            .propertyBuilder("sleeper.bulk.import.emr.serverless.release")
+            .description("The version of EMR Serverless to use.")
+            .propertyGroup(InstancePropertyGroup.BULK_IMPORT).defaultValue("emr-6.10.0")
+            .runCDKDeployWhenChanged(true).build();
+    UserDefinedInstanceProperty BULK_IMPORT_EMR_SERVERLESS_CUSTOM_IMAGE_REPO = Index
+            .propertyBuilder("sleeper.bulk.import.emr.serverless.repo")
+            .description(
+                    "The name of the repository for the EMR serverless container. The Docker image from the bulk-import module "
+                            + "should have been uploaded to an ECR repository of this name in this account.")
+            .propertyGroup(InstancePropertyGroup.BULK_IMPORT).runCDKDeployWhenChanged(true).build();
+    UserDefinedInstanceProperty BULK_IMPORT_EMR_SERVERLESS_TYPE = Index
+            .propertyBuilder("sleeper.bulk.import.emr.serverless.type")
+            .description("The type of EMR Serverless to use. Spark or Hive")
+            .propertyGroup(InstancePropertyGroup.BULK_IMPORT).defaultValue("Spark")
+            .runCDKDeployWhenChanged(true).build();
```

Comment:
This property could do with a validation predicate to ensure that only Spark or Hive options are allowed

Reply:
After talking with @gaffer01 I will remove this property